package.json

```diff
@@ -1,32 +1,36 @@
 {
   "name": "@metamask/module-template",
-  "version": "1.0.0",
-  "description": "",
+  "version": "0.0.0",
+  "description": "The MetaMask Node module template.",
   "publishConfig": {
     "registry": "https://registry.npmjs.org/",
     "access": "public"
   },
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.es.js",
-  "types": "dist/index.d.ts",
```

Comment:
No more types? :thinking: This is recommended by TypeScripts docs: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

Reply:
Whoops, fat-fingered that one. Restored!